lightning/src/routing/scoring.rs

```diff
@@ -253,7 +253,9 @@ impl<F> DynamicPenaltyScorer<F>
 where
 	F: Fn(u64, u64, u64, &NodeId, &NodeId) -> u64,
 {
-	/// Creates a new scorer using `penalty_func`.
+	/// Creates a new scorer that applies a closure `penalty_func`, which takes `short_channel_id:
```

Comment:
Not sure if there is a better way to document what the arguments of the closure should be, except spelling them out in this way?